target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Tue Jan 12 12:14:54 EET 2021
+version=1.0-SNAPSHOT
+groupId=com.murabi
+artifactId=com.murabi.java
```

Comment:
[major] Don't commit anything from the target dir since it contains only items that are built by maven.